get-reposense.py

```diff
@@ -16,12 +16,32 @@ def parse_args():
     group.add_argument('-r', '--release', action='store_true', help='Get RepoSense.jar from the latest release (Stable)')
     group.add_argument('-m', '--master', action='store_true', help='Get RepoSense.jar from the latest master (Beta)')
     group.add_argument('-t', '--tag', help='Get RepoSense.jar of a specific release version tag')
+    group.add_argument('-l', '--latest', help='Get RepoSense.jar of the latest release of a specific version tag')
     group.add_argument('-c', '--commit', help='Get RepoSense.jar of a specific commit')
 
     parser.add_argument('-o', '--overwrite', action='store_true', help='Overwrite RepoSense.jar file, if exists. Default: false')
 
     return parser.parse_args()
 
+def handle_latest_tag(tag):
+    page = 1
+    while True:
+        response = requests.get(f'https://api.github.com/repos/reposense/RepoSense/releases?per_page=100&page={page}')
+        if response.status_code in [403, 500]:
+            print('GitHub API has exceed the rate limit.')
+            exit(1)
+        releases = response.json()
+        if not releases:
+            print('Error, the provided tag does not exist!')
+            exit(1)
+        for i in releases:
+            release_tag = i['tag_name']
+            if release_tag[:len(tag)] == tag and \
+                    not (release_tag[len(tag):len(tag)+1].isdigit() and tag[-1].isdigit()):
```

Comment:
Nit: some tags look like `v1.9rc` which could match with `v1.9` depending on the order of releases but are not technically official releases - in general, release versions could include non-numeric characters. 

Maybe we can split by `.` and compare each section (or something else that makes sense)

Reply:
Hi, thanks for the review, I've updated it accordingly